awslambda/getsnippet/index.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+console.log('Loading function');
+
+const aws = require('aws-sdk');
+
+const s3 = new aws.S3({ apiVersion: '2006-03-01' });
+
+
+exports.handler = (event, context, callback) => {
+    console.log(JSON.stringify(event));
+    var user_id = event.pathParameters.user_id;
+    var body = JSON.parse(event.body);
+    var snippetKey = body.snippetTitle;
+    console.log(snippetKey);
+
+    snippetKey = snippetKey.replace(/\s+/g, '_').toLowerCase();
```

Comment:
Is this to sanitize the input? Should probably be its own function, or at least be commented.